roles/KubevirtCommonTemplatesBundle/defaults/main.yml

```diff
@@ -1,4 +1,2 @@
 ---
 # defaults file for KubevirtCommonTemplatesBundle
-version: "{{ templates_version |default('v0.6.2') }}"
```

Comment:
This has to stay here. The CR and the playbook use the `version` field and the default here is used when the user omits it.

Reply:
ack, templates are special enough.